packages/cloudflare/src/api/d1-tag-cache.ts

```diff
@@ -0,0 +1,173 @@
+import { debug, error } from "@opennextjs/aws/adapters/logger.js";
+import type { OpenNextConfig } from "@opennextjs/aws/types/open-next.js";
+import type { TagCache } from "@opennextjs/aws/types/overrides.js";
+import { RecoverableError } from "@opennextjs/aws/utils/error.js";
+
+import { getCloudflareContext } from "./cloudflare-context.js";
+
+/**
+ * An instance of the Tag Cache that uses a D1 binding (`NEXT_CACHE_D1`) as it's underlying data store.
+ *
+ * **Tag/path mappings**
+ *
+ * Information about the relation between tags and paths is stored in a `tags` table that contains
+ * two columns; `tag`, and `path`. The table name can be configured with `NEXT_CACHE_D1_TAGS_TABLE`
+ * environment variable.
+ *
+ * This table should be populated using an SQL file that is generated during the build process.
+ *
+ * **Tag revalidations**
+ *
+ * Revalidation times for tags are stored in a `revalidations` table that contains two columns; `tags`,
+ * and `revalidatedAt`. The table name can be configured with `NEXT_CACHE_D1_REVALIDATIONS_TABLE`
+ * environment variable.
+ */
+class D1TagCache implements TagCache {
+  public readonly name = "d1-tag-cache";
+
+  public async getByPath(rawPath: string): Promise<string[]> {
+    const { isDisabled, db, table } = this.getConfig();
+    if (isDisabled) return [];
+
+    const path = this.getCacheKey(rawPath);
+
+    try {
+      const { success, results } = await db
+        .prepare(`SELECT tag FROM ${table.tags} WHERE path = ?`)
```

Comment:
Can you please update the PR to always use `JSON.stringify(tableName)`

Reply:
I did in each place before, but conico asked me to move all the JSON.stringifies to a single place, so I moved it to the part where the variable is defined. I'll move them all back.